### DISCUSSION TITLE
YimMenu settings

### DISCUSSION THREAD

**Discussion by eclipsefiy** [YimMenu/YimMenu#1362] [General]
I wanted to ask where are the settings saved when you inject the cheat, because I'd like to get the fresh settings that you get when you launch the .dll, but it seems that they are the same even if I redownload the .dll

**Comment by Primexz** [comment]
> I wanted to ask where are the settings saved when you inject the cheat, because I'd like to get the fresh settings that you get when you launch the .dll, but it seems that they are the same even if I redownload the .dll

%appdata%/BigBaseV2/settings.json
